src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt

```diff
@@ -214,7 +214,8 @@ class TaskDeploymentService {
             ".${configurationResolver.getStringConfig(ConfigKey.AWS_REGION)}.compute.internal",
             ".dataworks.dwp.gov.uk",
             "localhost",
-            "127.0.0.1"
+            "127.0.0.1",
+            ".ap.dataworks.dwp.gov.uk"
```

Comment:
Isn't this covered by .dataworks.dwp.gov.uk at line 215?

Reply:
Yes  it is. Thanks, I am removing it.